src/styles/main.scss

```diff
@@ -2,6 +2,10 @@ html {
   font-family: Roboto, sans-serif;
 }
 
+header {
```

Comment:
don't use tag name as selectors, set corresponding class for this element

Reply:
Okay ! Got it!